[2022-01-29 14:07:36,679] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-08-02T06:00:00+00:00 [queued]>
[2022-01-29 14:07:36,713] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-08-02T06:00:00+00:00 [queued]>
[2022-01-29 14:07:36,714] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:07:36,714] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-29 14:07:36,714] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:07:36,769] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): get_data> on 2019-08-02 06:00:00+00:00
[2022-01-29 14:07:36,774] {standard_task_runner.py:52} INFO - Started process 326 to run task
[2022-01-29 14:07:36,779] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'HomeWorkWeek2', 'get_data', 'scheduled__2019-08-02T06:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions1.py', '--cfg-path', '/tmp/tmp_ebbn6w_', '--error-file', '/tmp/tmpqumz1c84']
[2022-01-29 14:07:36,780] {standard_task_runner.py:77} INFO - Job 25: Subtask get_data
[2022-01-29 14:07:36,919] {logging_mixin.py:109} INFO - Running <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-08-02T06:00:00+00:00 [running]> on host d1e1fe005577
[2022-01-29 14:07:37,061] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=HomeWorkWeek2
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2019-08-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-08-02T06:00:00+00:00
[2022-01-29 14:07:37,063] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-29 14:07:37,064] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "helloworld!!"']
[2022-01-29 14:07:37,077] {subprocess.py:85} INFO - Output:
[2022-01-29 14:07:37,080] {subprocess.py:89} INFO - helloworld!!
[2022-01-29 14:07:37,081] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-29 14:07:37,145] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=HomeWorkWeek2, task_id=get_data, execution_date=20190802T060000, start_date=20220129T140736, end_date=20220129T140737
[2022-01-29 14:07:37,227] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 14:07:37,332] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-30 10:43:08,885] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-08-02T06:00:00+00:00 [queued]>
[2022-01-30 10:43:08,934] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-08-02T06:00:00+00:00 [queued]>
[2022-01-30 10:43:08,934] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 10:43:08,935] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-30 10:43:08,935] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 10:43:08,979] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): get_data> on 2019-08-02 06:00:00+00:00
[2022-01-30 10:43:08,986] {standard_task_runner.py:52} INFO - Started process 548 to run task
[2022-01-30 10:43:08,993] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'HomeWorkWeek2', 'get_data', 'scheduled__2019-08-02T06:00:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions1.py', '--cfg-path', '/tmp/tmp5078mq8u', '--error-file', '/tmp/tmpmzy1kfo0']
[2022-01-30 10:43:08,994] {standard_task_runner.py:77} INFO - Job 70: Subtask get_data
[2022-01-30 10:43:09,197] {logging_mixin.py:109} INFO - Running <TaskInstance: HomeWorkWeek2.get_data scheduled__2019-08-02T06:00:00+00:00 [running]> on host fc414ca9b482
[2022-01-30 10:43:09,360] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-01-30 10:43:09,432] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=HomeWorkWeek2
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2019-08-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-08-02T06:00:00+00:00
[2022-01-30 10:43:09,435] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-30 10:43:09,437] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-08.csv > /opt/***output_2019-08.csv']
[2022-01-30 10:43:09,454] {subprocess.py:85} INFO - Output:
[2022-01-30 10:45:03,571] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-30 10:45:03,638] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=HomeWorkWeek2, task_id=get_data, execution_date=20190802T060000, start_date=20220130T104308, end_date=20220130T104503
[2022-01-30 10:45:03,705] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-30 10:45:03,786] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
